### DISCUSSION TITLE
Does this package use PIL anywhere ?

### DISCUSSION THREAD

**Discussion by mat100payette** [pymupdf/PyMuPDF#1060] [Looking for help] [answered]
I'm looking for bad dependencies matches in my solution and well I'm struggling to find this package's deps. Is there a list of its dependencies for each wheel somewhere ?

**Comment by JorjMcKie** [comment, marked as answer]
It has no dependencies at all, with the exception of 3 methods:
* `Pixmap.pillowWrite`and `Pixmap.pillowData` (new names: `Pixmap.pil_save()` and `Pixmap.pil_tobytes`). Both are optional - everything works without Pillow.
* `Document.subset_fonts`, which needs **_fontTools_**, also optional.

**Reply by mat100payette** [reply]
Ok thank you very much :)
